docs/src/main/asciidoc/reactive-sql-clients.adoc

```diff
@@ -59,16 +70,32 @@ public class Fruit {
 }
 ----
 
+== Prerequisites
+
+{prerequisites-docker}
```

Comment:
```suggestion
:prerequisites-docker:
```

Reply:
(What you used displays the attribute, what I suggest sets its value)

Reply:
Sorry for that, and thank for reviewing